com.etlegacy.ETL.yaml

```diff
@@ -0,0 +1,239 @@
+app-id: com.etlegacy.ETL
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+branch: stable
+
+# 64bit is currently unfunctional. Upsteam is aware of this,
+# but there is no definitive solution as of yet.
+
+# https://dev.etlegacy.com/issues/1211
+
+# There is also a more controversial problem, relating to mods.
+# Many mods are made under a crappy non-GPL compatible SDK license which
+# means that those mods will forever remain closed source, 32bit only.
+# Good hope is that overtime they will die, but until then, we better stick
+# to 32bit
+
+rename-desktop-file: etlegacy.desktop
+rename-icon: etl
+
+command: etl
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Compat.i386
+  - org.freedesktop.Sdk.Extension.toolchain-i386
+
+finish-args:
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --allow=multiarch
+  - --persist=.etlegacy
+
+cleanup:
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+  - /share/gtk-doc
+  - /share/man
+  - /share/pkgconfig
+  - '*.a'
+  - '*.la'
+
+add-extensions:
+  org.freedesktop.Platform.Compat.i386:
+    directory: lib/i386-linux-gnu
+    version: '19.08'
+
+  org.freedesktop.Platform.Compat.i386.Debug:
+    directory: lib/debug/lib/i386-linux-gnu
+    version: '19.08'
+    no-autodownload: true
+    
+  org.freedesktop.Platform.GL32:
+    directory: lib/i386-linux-gnu/GL
+    version: '1.4'
+    versions: 19.08;1.4
+    subdirectories: true
+    no-autodownload: true
+    autodelete: false
+    add-ld-path: lib
+    merge-dirs: vulkan/icd.d;glvnd/egl_vendor.d;OpenCL/vendors;lib/dri;lib/d3d;vulkan/explicit_layer.d
+    download-if: active-gl-driver
+    enable-if: active-gl-driver
+    
+x-compat-i386-opts: &compat_i386_opts
+  prepend-pkg-config-path: /app/lib32/pkgconfig:/usr/lib/i386-linux-gnu/pkgconfig
+  ldflags: -L/app/lib32
+  prepend-path: /usr/lib/sdk/toolchain-i386/bin
+  env:
+    CC: i686-unknown-linux-gnu-gcc
+    CXX: i686-unknown-linux-gnu-g++
+  libdir: /app/lib32
+    
+modules:
+  
+  - name: glu
+    build-options:
+      arch:
+        x86_64: *compat_i386_opts
+    config-opts:
+      - --disable-static
+    sources:
+      - type: archive
+        url: https://mesa.freedesktop.org/archive/glu/glu-9.0.1.tar.xz
+        sha256: fb5a4c2dd6ba6d1c21ab7c05129b0769544e1d68e1e3b0ffecb18e73c93055bc
+
+  # Info concerning glew 32bit
+  # https://stackoverflow.com/questions/15859031/glew-1-9-0-builds-a-64-bit-so-even-with-m32-argument
+
+  - name: glew
+    build-options:
+      arch:
+        x86_64: *compat_i386_opts
+    no-autogen: true
+    config-opts:
+      - -DCMAKE_C_FLAGS=-m32 -L/usr/X11R6/lib -L/usr/lib
```

Comment:
These -L invocations look really suspicious. Both are quite likely incorrect for freedesktop-sdk. There's /usr/lib/{gcc-triplet}/, one per arch. Iirc x86 one is i386-linux-gnu